### PR TITLE
Correct RSD sort order and add test coverage

### PR DESCRIPTION
**Pull request by wgu-edwin**

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -301,6 +301,12 @@
             <version>1.11.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (JohnKallies) [inline api/pom.xml:304]: We should already have AssertJ available in the dependency tree. If we need to upgrade, then we probably need to upgrade Spring starter parent first.

File: api/src/test/kotlin/edu/wgu/osmt/richskill/RichSkillSortOrderTest.kt [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        override val collectionEsRepo: CollectionEsRepo,
+        override val keywordEsRepo: KeywordEsRepo,
+        override val jobCodeEsRepo: JobCodeEsRepo
+): SpringTest(), BaseDockerizedTest, HasDatabaseReset, HasElasticsearchReset {
+
+    @Autowired
+    lateinit var richSkillController: RichSkillController
+
+    @Autowired
+    lateinit var batchImportRichSkill: BatchImportRichSkill
```
Comment (JohnKallies) [inline api/src/test/kotlin/edu/wgu/osmt/richskill/RichSkillSortOrderTest.kt:35]: I don't think this is used here

File: api/pom.xml (hunk 1 above)
Comment (jchavez137) [reply api/pom.xml]: Removed

File: api/src/test/kotlin/edu/wgu/osmt/richskill/RichSkillSortOrderTest.kt (hunk 2 above)
Comment (jchavez137) [reply api/src/test/kotlin/edu/wgu/osmt/richskill/RichSkillSortOrderTest.kt]: Removed

**Review by JohnKallies — APPROVED** [review]

**Pull request merged**